src/identity/storage/PassthroughKeyValueStore.ts

```diff
@@ -0,0 +1,72 @@
+import { BasicRepresentation } from '../../ldp/representation/BasicRepresentation';
+import type { Representation } from '../../ldp/representation/Representation';
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import type { ResourceStore } from '../../storage/ResourceStore';
+import { APPLICATION_OCTET_STREAM } from '../../util/ContentTypes';
+import { readableToString } from '../../util/StreamUtil';
+import type { KeyValueStore } from './KeyValueStore';
+
+export interface StoredData {
+  payload: unknown;
+  expires?: string;
+}
+
+/**
+ * A KeyValueStore that wraps a provided ResourceStore
+ */
+export class PassthroughKeyValueStore implements KeyValueStore {
```

Comment:
```suggestion
export class KeyValueResourceStore implements KeyValueStore {
```

Reply:
Again, I disagree. This is not a ResourceStore. It's a Key Value Store. Calling it a ResourceStore is false advertising.